aspnetcore/security/authentication/individual.md

```diff
@@ -16,7 +16,6 @@ The authentication templates are available in .NET Core CLI with `-au Individual
 
 ```console
 dotnet new mvc -au Individual
-dotnet new webapi -au Individual
```

Comment:
It was there for both 2.0 and 2.1 ... it dropped out as a template at 2.2 AFAICT from looking at the 2.1 template NuGet package.

<img width="458" alt="capture" src="https://user-images.githubusercontent.com/1622880/53206080-a8fe2b80-35f5-11e9-9891-9fba6cec1359.PNG">

Reply:
... or maybe not. You'll have to ask them. I'm not sure when they dropped it.

<img width="484" alt="capture" src="https://user-images.githubusercontent.com/1622880/53206290-35105300-35f6-11e9-9fb1-379edaf1cd39.PNG">

Reply:
Oh, I see it's not the template ... it's the Individual auth option that was dropped.